site/content/docs/user/kind-example-config.yaml

```diff
@@ -1,30 +1,29 @@
 # this config file contains all config fields with comments
-kind: Config
-apiVersion: kind.sigs.k8s.io/v1alpha2
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+# patch the generated kubeadm config with some extra settings
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta1
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  networking:
```

Comment:
v1alpha3 is going to have a new `networking` field #340 .

I've sent a proposal (#348)  using ` serviceSubnet` and `podSubnet` in the new `networking` section.

Does it make sense?

Reply:
depends on what we want in the kind example config file(s).

Reply:
Let's only add what is merged for now with the API server options and come back to discussing these other options (next meeting / 0.3 planning perhaps?)

Reply:
ok, i'd prefer if we talk about this PR the next meeting and see if this config is what we want for the next release.

Reply:
SGTM, 0.2 is about ready to go, would be good to document v1alpha3 for that and add in these extra fields later.

Reply:
.2 mostly needs relnotes and https://github.com/kubernetes-sigs/kind/pull/394 https://github.com/kubernetes-sigs/kind/pull/393

Reply:
so is this PR good for 0.2?